docs/src/api_bindings.md

```diff
@@ -169,37 +169,97 @@ h5e_walk
 ---
 
 ## [[`H5F`](https://portal.hdfgroup.org/display/HDF5/Files) — File Interface](@id H5F)
```

Comment:
Presumably these were all determined using the generator you have developed?  Thus I haven't reviewed them for correctness, since I think the generator has been working without issues.

Reply:
Yes they are coming from https://github.com/mkitti/LibHDF5.jl/blob/712b6e306a15de37f748727b37676aca70ea0664/src/bind.jl

That is generated by this code:
https://github.com/mkitti/LibHDF5.jl/blob/main/gen/bind_generator.jl